port/common/omrportcontrol.c

```diff
@@ -72,8 +72,39 @@ omrport_control(struct OMRPortLibrary *portLibrary, const char *key, uintptr_t v
 			return (int32_t)PPG_mem_mem32_subAllocHeapMem32.suballocator_commitSize;
 		}
 		return 0;
+	} else if (!strcmp(OMRPORT_CTLDATA_ALLOCATE32_INCREMENT_SIZE, key)) {
```

Comment:
Please use `0 ==` instead of `!`.

Reply:
I think I should also update the one on line 57 while I am here.